src/intTest/resources/edifact/pathology_IAF_regex.nhsack.dat

```diff
@@ -0,0 +1,9 @@
+^UNA:\+.\?\s'
+UNB\+UNOC:3\+TES5\+XX11\+([0-9]+):([0-9]+)\+([0-9]+)\+\+PATH_MEDRPT_V3_ACK'
+UNH\+1\+NHSACK:D:1:NH:NHS001'
+BGM\+AKR'
+DTM\+137:([0-9]+):102'
+NIR\+SIR:1015\+XX11\+TES5\+IAF'
+NMR\+AGO:1\+MA'
+UNT\+6\+1'
+UNZ\+1\+([0-9]+)'
```

Comment:
While we're using regexes, let's keep it as clean as possible. `[0-9]` = `\d`, you don't need `(capture groups)` if you're not inspecting the captured values, and you've `^anchored` the start but not the `end$`
```suggestion
^UNA:\+.\?\s'
UNB\+UNOC:3\+TES5\+XX11\+\d+:\d+\+\d+\+\+PATH_MEDRPT_V3_ACK'
UNH\+1\+NHSACK:D:1:NH:NHS001'
BGM\+AKR'
DTM\+137:\d+:102'
NIR\+SIR:1015\+XX11\+TES5\+IAF'
NMR\+AGO:1\+MA'
UNT\+6\+1'
UNZ\+1\+\d+'$
```

Reply:
Updated the regex as requested.